src/Workspaces/Core/Portable/Classification/SyntaxClassification/AbstractSyntaxClassificationService.cs

```diff
@@ -53,5 +53,8 @@ protected AbstractSyntaxClassificationService()
         {
             Worker.Classify(workspace, semanticModel, textSpan, result, getNodeClassifiers, getTokenClassifiers, cancellationToken);
         }
+
+        public async Task<TextChangeRange?> ComputeSyntacticChangeRangeAsync(Document oldDocument, Document newDocument, CancellationToken cancellationToken)
+            => await SyntacticChangeRangeComputer.ComputeSyntacticChangeRangeAsync(oldDocument, newDocument, cancellationToken).ConfigureAwait(false);
```

Comment:
async/await needed here as the helper returns the non-nullable version.

Reply:
No need for async/await. You can use the `AsNullable()` extension method to cast `Task<T>` to `Task<T?>` without warnings.